RadixWallet/Features/AssetsFeature/Components/HelperViews/ResourceBalance/ResourceBalance+Helpers.swift

```diff
@@ -88,7 +88,7 @@ extension ResourceBalance: Comparable {
 // MARK: - ResourceBalance.Amount + Comparable
 extension ResourceBalance.Amount: Comparable {
 	public static func < (lhs: Self, rhs: Self) -> Bool {
-		// If RETDecimal were comparable:
+		// If Decimal192 were comparable:
```

Comment:
REVERT / FIX we can make them comparable!

Reply:
If you add that, you can uncomment this bit and remove the comment!

Reply:
TODO: Upgrade this code since Decimal192 ARE Comparable :D